doc/tensor.md

```diff
@@ -1327,36 +1331,64 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] torch.gather(src, dim, index) ###
```

Comment:
Yes, that's a bit confusing. You can actually do all of the following:
```lua
torch.gather(src, dim, idx)  -- Writes the result to a new tensor.
src:gather(dim, idx)         -- Same as above.
dst:gather(src, dim, idx)    -- Writes the result to dst.
```
This isn't the case for some other functions, but I'm not sure whether the `torch.` helps here.